erc20-connector/contracts/AuroraERC20Locker.sol

```diff
@@ -0,0 +1,164 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@openzeppelin/contracts/utils/math/SafeMath.sol";
+import "@openzeppelin/contracts/utils/Strings.sol";
+import "rainbow-bridge-sol/nearbridge/contracts/AdminControlled.sol";
+import "./AccountIds.sol";
+
+contract AuroraERC20Locker is AdminControlled, AccountIds {
```

Comment:
Is there any objection to having this contract inside `erc20-connector` folder with others? I don't think so and IMO it's better to keep it in a separate directory, especially as there are no contract intersections.

Reply:
Fixed at 2271dc717d43bcbd80d1ea824a15211039fbd2e8